internal/schema-registry/command_kek_list.go

```diff
@@ -0,0 +1,58 @@
+package schemaregistry
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/config"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type kekListOut struct {
+	Name string `human:"Name" serialized:"name"`
```

Comment:
We really don't want to add any more fields here?

Reply:
🌴 added all fields now...